resources/skins/nudus-dark/metadata.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<skin version="0.02" base="nudus-base">
+  <author>
+    <name>akinokonomi</name>
+  </author>
+  <palette>
+    <color key="FgInteresting"          >#85ACF6</color>
```

Comment:
This is some of the most bizzare XML formatting I've ever seen. Not saying it's good or bad, just unusual.

Reply:
Yeah, this is the first time I'm doing such thing, too... But I've seen something similar somewhere before. 

It remains quite readable while looking more consistent. I would not say this is balanced, but it's close to be.
Any suggestions are welcome too.